sahaget_bn/urls.py

```diff
@@ -16,7 +16,24 @@
 """
 from django.contrib import admin
 from django.urls import path, include  # new
+from drf_spectacular.views import (
+    SpectacularAPIView,
+    SpectacularRedocView,
+    SpectacularSwaggerView,
+)
 
 urlpatterns = [
     path("admin/", admin.site.urls),
+    path("api/v1/users/", include("accounts.urls")),
+    path("api/v1/schema/", SpectacularAPIView.as_view(), name="schema"),
+    path(
+        "api/v1/schema/redoc/",
+        SpectacularRedocView.as_view(url_name="schema"),
+        name="redoc",
+    ),
+    path(
+        "api/v1/schema/swagger-ui/",
```

Comment:
if this is the endpoints for swagger docs renew it to /api/v1/api-docs

Reply:
@usanaphtal112

Reply:
Resolved!